spec/models/rubric_criterion_spec.rb

```diff
@@ -16,6 +17,55 @@
     it { is_expected.to validate_presence_of(:levels) }
     it { is_expected.to have_one(:course) }
 
+    context 'Helper method #update_levels' do
```

Comment:
This is actually a good time to use a `describe` block (even though the rest of this file doesn't seem to). Move this code outside of the outer context, and use `describe #update_levels` instead.

Also based on my above feedback, don't check the return values; instead, check whether the method properly updates the associated level objects or not.